packages/flutter_blue_plus_platform_interface/lib/src/bluetooth_msgs.dart

```diff
@@ -80,6 +80,7 @@ class BmScanSettings {
   final bool androidLegacy;
   final int androidScanMode;
   final bool androidUsesFineLocation;
+  final List<Guid> webOptionalServices;
```

Comment:
I would suggest making this nullable (or default to an empty list) to cover other non-web platforms.

```suggestion
  final List<Guid>? webOptionalServices;
```

Reply:
I like using an empty list